packages/components/tests/integration/components/hds/tabs/index-test.js

```diff
@@ -96,4 +96,28 @@ module('Integration | Component | hds/tabs/index', function (hooks) {
       .doesNotHaveAttribute('aria-selected');
     assert.dom('[data-test="NOT selected panel"]').hasAttribute('hidden');
   });
+
+  test('it should render an icon when passed into a tab', async function (assert) {
+    await render(hbs`
+      <Hds::Tabs as |T|>
+        <T.Tab @icon="info">One</T.Tab>
+        <T.Tab>Two</T.Tab>
+        <T.Panel>Content 1</T.Panel>
+        <T.Panel>Content 2</T.Panel>
+      </Hds::Tabs>
+    `);
+    assert.dom('.hds-tabs__tab-icon').exists();
+  });
+
+  test('it should render a count when passed into a tab', async function (assert) {
+    await render(hbs`
+      <Hds::Tabs as |T|>
+        <T.Tab @count="5">One</T.Tab>
+        <T.Tab>Two</T.Tab>
+        <T.Panel>Content 1</T.Panel>
+        <T.Panel>Content 2</T.Panel>
+      </Hds::Tabs>
+    `);
+    assert.dom('.hds-tabs__tab-count').exists();
+  });
```

Comment:
Added these basic tests for passed in "icon" and "count". Let me know if you prefer that I check for these using the class names instead of `exists()` or any improvements I should make.

Reply:
they work for me.